server/lib/src/schema.rs

```diff
@@ -449,15 +450,20 @@ impl SchemaClass {
             .get_ava_single_iutf8(Attribute::ClassName)
             .map(AttrString::from)
             .ok_or_else(|| {
-                admin_error!("missing {} - {:?}", Attribute::ClassName, uuid);
+                error!("missing {} - {:?}", Attribute::ClassName, uuid);
                 OperationError::InvalidSchemaState(format!("missing {}", Attribute::ClassName))
             })?;
+
+        if name == "key_provider" {
+            trace!(?value);
+        }
```

Comment:
Since we guard trace behind debug/test, this probably should be too, to avoid the check?

Reply:
Or just remove it 😄

Reply:
Yeah, it was a leftover debugging statement.